src/tournament.cpp

```diff
@@ -1573,18 +1634,22 @@ void Tournament::AddMatchTable(MatchTable* NewMatchTable)
 
 void Tournament::OnMatchStarted(const Match& Match) const
 {
-	ScheduleSave();
 	if (m_Application)
 		m_Application->RequestPushToResultsServer();
+
+	ScheduleSave();

```

Comment:
in preparation tag should be added here, also for the next match!
On has to add a function a find the next match, given the match that has already started.